performance/implementations/glamor/Box/index.js

```diff
@@ -6,12 +6,7 @@ import View from '../View';
 const Box = ({ color, fixed = false, layout = 'column', outer = false, ...other }) => (
   <View
     {...other}
-    style={[
-      styles[`color${color}`],
-      fixed && styles.fixed,
-      layout === 'row' && styles.row,
-      outer && styles.outer
-    ]}
+    style={[styles[`color${color}`], fixed && styles.fixed, layout === 'row' && styles.row, outer && styles.outer]}
```

Comment:
@vjeux @jlongster this formatting change is unfortunate for RN styles / components. Thinking out loud, I wonder if proxies for reading complexity of code – e.g., use of conditionals and multiple elements – could eventually be used for context-specific formatting.